agent-install/agent-install.sh

```diff
@@ -824,6 +824,7 @@ function store_cert_file_permanently() {
 # For both device and cluster: Returns true (0) if /etc/default/horizon already has these values
 function is_horizon_defaults_correct() {
     log_debug "is_horizon_defaults_correct() begin"
+    IS_HORIZON_DEFAULTS_CORRECT=1
```

Comment:
I'd like you to change this value to `'true'` or `'false'` . When i look at this i don't know if `1` means it is correct or is not correct.